db/migrate/20240224005738_create_entities.rb

```diff
@@ -0,0 +1,12 @@
+class CreateEntities < ActiveRecord::Migration[7.1]
+  def change
+    create_table :entities do |t|
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=NOTE%3A%20do%20not%20change%20column%20names%20visible%20in%20the%20diagram.%20You%20need%20to%20change%20the%20%22Entity%22%20name%20according%20to%20the%20theme%20you%20have%20chosen%20for%20your%20project%20(please%20note%3A%20%22Transaction%22%20is%20a%20name%20already%20used%20by%20ActiveRecord%2C%20so%20using%20it%20as%20a%20name%20for%20your%20model%20and%20table%20will%20result%20in%20an%20error).), please change the "Entity" name according to the theme you have chosen for your project. You can use `payment` instead, kindly don't forget to change it for table name, controller, model, views, and routes to make everything work as expected.